app/controllers/cards_controller.rb

```diff
@@ -1,5 +1,69 @@
 class CardsController < ApplicationController
+  before_action :set_card, only: [:show, :edit, :update, :destroy]
+
   def index
     @cards = Card.all
+
+    respond_to do |format|
+      format.html
+    end
   end
+
+  def show
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new
+    @card = Card.new
+
+    respond_to do |format|
+      format.html
+    end
+  end
+
+
+  def edit
+  end
+
+  def create
+    @card = Card.new(card_params)
+
+    respond_to do |format|
+      if @card.save
+        format.html { redirect_to @card, notice: t('card.created')  }
+      else
+        format.html { render action: "new" }
+      end
+    end
+  end
+
+  def update
+    respond_to do |format|
+      if @card.update(card_params)
+        format.html { redirect_to @card, notice: t('card.updated') }
+      else
+        format.html { render action: "edit" }
+      end
+    end
+  end
+
+  def destroy
+    respond_to do |format|
+      @card.destroy
+      format.html { redirect_to cards_url, notice: t('card.deleted') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.